src/Accordion.php

```diff
@@ -85,18 +76,23 @@ class Accordion extends Widget
      * ])
      * ```
      */
-    public $items = [];
+    private $items = [];
+
     /**
      * @var bool whether the labels for header items should be HTML-encoded.
      */
-    public $encodeLabels = true;
+    private $encodeLabels = true;
+
     /**
-     * @var bool whether to close other items if an item is opened. Defaults to `true` which causes an
-     * accordion effect. Set this to `false` to allow keeping multiple items open at once.
+     * @var bool whether to close other items if an item is opened. Defaults to `true` which causes an accordion effect.
+     *           Set this to `false` to allow keeping multiple items open at once.
      */
-    public $autoCloseItems = true;
+    private $autoCloseItems = true;
+
     /**
-     * @var string the HTML options for the item toggle tag. Key 'tag' might be used here for the tag name specification.
+     * @var array the HTML options for the item toggle tag. Key 'tag' might be used here for the tag name
+     *             specification.
```

Comment:
```suggestion
     * specification.
```

Reply:
Do we have indents in all the packages in which the variables are defined, are they going to be removed?

Reply:
Do we have indents in all the packages in which the variables are defined, are they going to be removed?